cobigen-cli/class-loader-agent/pom.xml

```diff
@@ -10,12 +10,12 @@
     <version>dev-SNAPSHOT</version>
   </parent>
   
-  
   <build>
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>3.1.2</version>
```

Comment:
is this a newer version? if so maybe you can upgrade it on the master pom to not let every plugin overwrite it again on it's own